smartsim/experiment.py

```diff
@@ -168,11 +168,21 @@ def __init__(
         if self._launcher == "cobalt":
             raise SSUnsupportedError("Cobalt launcher is no longer supported.")
 
+        if launcher == "dragon":
+            self._set_dragon_server_path()
+
         self._control = Controller(launcher=self._launcher)
 
         self.db_identifiers: t.Set[str] = set()
         self._telemetry_cfg = ExperimentTelemetryConfiguration()
 
+    def _set_dragon_server_path(self) -> None:
+        """Set path for dragon server through environment varialbes"""
```

Comment:
should we be using `config.dragon_server_path` to keep the env var name in one place?

Reply:
Do we really want to introduce a second env var here?